### DISCUSSION TITLE
Welcome to community Discussions!

### DISCUSSION THREAD

**Discussion by Peefy** [kcl-lang/community#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mehemken** [comment]
Hello!
I'm Marco. I work on a platform team and I'm interested in improving how I manage YAML. Looking forward to learning more about KCL.

**Reply by Peefy** [reply]
Hello. Welcome to the KCL community. ❤

**Comment by markermurt** [comment]
Bonjour à tous, ravi d'être parmi vous. Je rejoins cette communauté pour apprendre, échanger des idées et rencontrer des personnes réellement intéressées par le sujet. J'aime poser des questions et approfondir les sujets, surtout lorsque cela peut être utile à d'autres. Je suis également heureux de partager mes propres expériences si elles peuvent s'avérer pertinentes. Pour moi, une bonne communauté repose sur des discussions respectueuses, la curiosité et l'entraide. J'ai hâte de faire votre connaissance et de participer activement aux discussions. Une opportunité idéale [Betblast casino](https://betblast-france.fr/) pour tester les jeux phares du casino.
